server/channels/app/platform/config.go

```diff
@@ -326,12 +328,18 @@ func (ps *PlatformService) LimitedClientConfig() map[string]string {
 }
 
 func (ps *PlatformService) IsFirstUserAccount() bool {
-	count, err := ps.Store.User().Count(model.UserCountOptions{IncludeDeleted: true})
-	if err != nil {
-		return false
+	if fetchUserCount {
```

Comment:
Maybe to avoid the confusion of returning `fetchUserCount` we could just do `return false` for the base case and add a `return false` after assigning the variable and `return true` at the bottom of the fetch user count block. 
0/5 I think the code works as is.